arm/capture.c

```diff
@@ -7,10 +7,12 @@
 #include <asoundlib.h>
 
 #include "capture.h"
-#include "freq.h"
 
 
 #define EXCRUCIATING_DETAIL 0
+#define LATENCY 10e-3f
+// It would just be stupid to run with a period any lower than this.
```

Comment:
Can you elaborate on this, rather than just saying stupid?

Reply:
Or I could just nuke it. A somewhat high latency automatically forbids ridiculous capture periods.